tools/libfrencutils/affinity.c

```diff
@@ -127,5 +144,4 @@ int set_cpu_affinity(int cpu)
 #endif
   return 0;
 }
-
-int set_cpu_affinity_(int *cpu) { return set_cpu_affinity(*cpu); }	/* Fortran interface */
```

Comment:
Is this deleted function needed?

Reply:
> Is this deleted function needed?

I rechecked. Only the set_cpu_affinity  function on line 134 is used.